librosa/core/constantq.py

```diff
@@ -131,9 +131,9 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
     n_octaves = int(np.ceil(float(n_bins) / bins_per_octave))
 
     # Make sure our hop is long enough to support the bottom octave
-    if hop_length < 2**n_octaves:
-        raise ParameterError('Insufficient hop_length {:d} '
-                             'for {:d} octaves'.format(hop_length, n_octaves))
+    if np.mod(hop_length, 2**n_octaves) != 0 or hop_length < 2**n_octaves:
```

Comment:
Yes, `hop_length == 0` or `hop_length == - 2**n_octaves` would both pass if not for the second check.